ui/.env.local.default

```diff
@@ -24,3 +24,4 @@ FROM_EMAIL="fake@example.com"
 #STRIPE_YEARLY_PLAN_PRICE_ID
 
 #TERMS_UPDATED_AT="2022-05-30T20:18:35.225Z"
+#ERROR_REPORTING_WEBHOOK=
```

Comment:
what should we set this to in prod?

Reply:
A new webhook should be created on discord. In case of no webhook, there will be no error message in discord

Reply:
Why on discord? Wouldn't that very quickly become unmanageable? But I guess we could maybe try it as a first iteration

Reply:
added now